service/manager.go

```diff
@@ -85,6 +86,10 @@ func newManager(builder *Builder) (Manager, error) {
 		return nil, err
 	}
 
+	if err := m.configProvider.Get(config.Root).Populate(m); err != nil {
```

Comment:
builder produces only one service, so there could be multiple services, but they need to be created differently.